plugin/dadbod.vim

```diff
@@ -27,7 +27,7 @@ call extend(g:dbext_schemes, {
       \ }, 'keep')
 
 command! -bang -nargs=? -range=-1 -complete=custom,db#command_complete DB
-      \ exe db#execute_command(<bang>0, <line1>, <count>, substitute(<q-args>,
+      \ exe db#execute_command(<q-mods>, <bang>0, <line1>, <count>, substitute(<q-args>,
```

Comment:
Change this to `'<mods>'`, and add `let mods = a:mods ==# '<mods>' ? '' : a:mods` to enable this to work with older versions of Vim.

Reply:
Cool, didn't know about that. Just a sec.

Reply:
Older versions don't support . . . what? The `q-` modifier? (Just out of curiosity.)

Reply:
It doesn't support `<mods>` in any form. `'<mods>'` (don't forget the quotes) causes it to pass a literal 6 character string, which we then check for and ignore in that ternary assignment.